semantics/src/main/kotlin/com/strumenta/kolasu/semantics/symbol/importer/SymbolImporter.kt

```diff
@@ -0,0 +1,20 @@
+package com.strumenta.kolasu.semantics.symbol.importer
+
+import com.strumenta.kolasu.model.Node
+import com.strumenta.kolasu.model.children
+import com.strumenta.kolasu.semantics.symbol.provider.SymbolProvider
+import com.strumenta.kolasu.semantics.symbol.repository.SymbolRepository
+
+open class SymbolImporter(
```

Comment:
What is the role of a SymbolImporter? Does it translate nodes into SymbolDescriptions?

Reply:
Exactly, the `SymbolImporter` can be used to transform an AST into its corresponding `SymbolDescription` instances and store these into the `SymbolRepository` instance.

Reply:
It could be useful to add a very simple comment to explain this